admin-site/src/App.svelte

```diff
@@ -189,38 +198,50 @@
   let indexInOccasions;
 
   let sliderCue;
-  let broadcastStatus = "unsent"
-  let broadcastResults
+  let broadcastStatus = "unsent";
+  let broadcastResults;
 
 //password check on login
   let authenticated = false;
 
 //to show/hide dev Tools.
   let devElement = false;
 
+//show/hide modal
+  // let modalhtml;
+  //double check that a delete Occasion has happened
+  let deleteOccasionHasHappened = false;
+
+  //hold DOM state
+  let pageState = 0;
```

Comment:
I'm down for this approach for now, it makes it easier to understand the different states. Routing is the next step...